### DISCUSSION TITLE
[SOLVED] qBittorrent high CPU usage when using I2P (I2PD)

### DISCUSSION THREAD

**Discussion by absolutep** [PurpleI2P/i2pd#2128] [General]
This solution is for people who are using I2P with qBittorrent and facing high CPU usage.

This applies for both Mixed mode & I2P-only mode.

**Solution - Just remove all public trackers from your torrents.**

Solution source as per image attached
![Untitled](https://github.com/user-attachments/assets/af37b38f-6726-477c-8589-f28a420d02b0)

**Comment by anikey-from-i2p** [comment]
I'm thinking this is more of an issue of qbittorrent, so you should have posted it on qbittorrent discussion instead.

Or it's more likely a bug, so maybe even file an issue.

**Reply by absolutep** [reply]
> I'm thinking this is more of an issue of qbittorrent, so you should have posted it on qbittorrent discussion instead.
> 
> Or it's more likely a bug, so maybe even file an issue.

Issue is already present in libtorrent = https://github.com/arvidn/libtorrent/issues/7535

I have posted it in qBittorrent discussion as well, let's see what happens.

I mean for me, everything is open, so I can easily connect to all Public leechers/seeders via DHT properly without any issues.

That may not be the case for other people, who need public trackers to connect.

**Reply by anikey-from-i2p** [reply]
Please note that QBittorrent does not (as of now) support DHT in I2P.

**Reply by absolutep** [reply]
> Please note that QBittorrent does not (as of now) support DHT in I2P.

I was referring to public DHT and not I2P DHT.
